config/storage/production.yml

```diff
@@ -0,0 +1,16 @@
+# For case notes so far, the default bucket option
+amazon_case_notes:
```

Comment:
I think we should use a better name, like `amazon_internal_bucket` for this one and `amazon_public_bucket` for the other.

Reply:
Done.